src/urllib3/util/ssl_.py

```diff
@@ -320,6 +328,16 @@ def create_urllib3_context(
 
     context.options |= options
 
+    if verify_flags is None:
+        verify_flags = 0
+        # In python 3.13+ ssl.create_default_context() sets VERIFY_X509_PARTIAL_CHAIN
+        # and VERIFY_X509_STRICT so we do the same
+        if sys.version_info >= (3, 13):
+            verify_flags |= VERIFY_X509_PARTIAL_CHAIN
+            verify_flags |= VERIFY_X509_STRICT
+
+    context.verify_flags |= verify_flags
```

Comment:
It seems passing `verify_flags` to `create_urllib3_context` provides no way to set `verify_flags` to 0 because `|=` is used here, right?

Reply:
You're right. I took the same approach as `options`. There is a default provided by `SSLContext(PROTOCOL_TLS_CLIENT)` (which is to set `VERIFY_X509_TRUSTED_FIRST`) and we just add flags to it. If a user want to clear the default (unset `VERIFY_X509_TRUSTED_FIRST`) this can only be done by excplicitly setting `verify_flags` after calling `create_urllib3_context`. The same goes for `options`.